.coderabbit.yaml

```
@@ -21,38 +21,52 @@ reviews:
   related_issues: true
   related_prs: true
   suggested_labels: true
-  auto_apply_labels: false
+  auto_apply_labels: true
   suggested_reviewers: true
-  auto_assign_reviewers: false
+  auto_assign_reviewers: true
   poem: true
-  labeling_instructions: []
+  labeling_instructions: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)